### ISSUE TITLE
Import widget is missing

### ISSUE THREAD

**Issue by ftastisch** [biolab/orange3-text#495]
##### Text version
Will check asap

##### Orange version
3.24.1

##### Expected behavior
Be able to import text documents.

##### Actual behavior
Import widget is missing.

##### Steps to reproduce the behavior
install orange3 and orange3-text

**Comment by ajdapretnar** [comment]
Please also describe how you installed Orange (conda, pip, installer)? And how you installed the add-on? Via the add-on dialogue?

Also add you OS info.

**Comment by ftastisch** [comment]
Hi! I had the problem both with installation via stand-alone (Windows 10) and conda (Windows 10, Manjaro). Installation of orange-text was done via addon-dialogue (standalone installation) and conda (conda installations).

**Comment by ajdapretnar** [comment]
Right, I have experienced this issue before. For some (unknown) reason, the add-on dialogue doesn't install the latest version of Text add-on, but 0.7.1. This version is missing some is incompatible with the current Orange.

**Solution:**
Go to add-on dialogue.
You should see the option to upgrade Orange3-Text to 0.7.4.
Upgrade and restart Orange.
The widget should appear.

**Comment by ftastisch** [comment]
Thanks, that helped and I'm able to import again in the standalone installation. I'll check tonight whether this works also in the conda installation, but I think I remember that the installation via the dialogue was not possible in this case.

**Comment by ajdapretnar** [comment]
I think conda install orange3-text=0.7.4 should work.

**Comment by ftastisch** [comment]
> I think `conda install orange3-text=0.7.4` should work.

Thanks, that worked!

**Comment by vippk0000** [comment]
I am also facing the same issue. I just installed a fresh version of Orange via standalone installer and then added all widgets via the add-on dialogue. But the import document widget is missing. 

Tried all above given solutions, even installed orange via Anaconda but the import doc option is still unavailable. 

Using Orange 3.25.0
Text version 0.9.1
OS Windows 10 Pro

Another interesting problem is that when I click on Add-ons... option, it throws an error : There is an issue with internet connection ( image attached), but my internet is perfectly fine. It also shows me Add-ons dialog box empty when i close the warning box.

![error_import doc](https://user-images.githubusercontent.com/65536161/82194805-024a2d00-9915-11ea-93fb-51f2b2d86ef0.PNG)
